src/cross_module_fn.c

```diff
@@ -134,8 +134,13 @@ error_hypercore_proxy_index_options(Datum reloptions, bool validate)
  * parsing index options instead.
  */
 static Datum
-error_pg_community_hypercore_proxy_handler(PG_FUNCTION_ARGS)
+process_hypercore_proxy_handler(PG_FUNCTION_ARGS)
 {
+	ts_license_enable_module_loading();
+
+	if (ts_cm_functions->hypercore_proxy_handler != process_hypercore_proxy_handler)
+		return ts_cm_functions->hypercore_proxy_handler(fcinfo);
+
```

Comment:
Can we test this? Set community license and then try to vacuum indexes, for example?